src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { Greeting } from "./greeting";
+
+export const greeting: Greeting = {
+    greetee: "World",
```

Comment:
This will change as the interface from `greeting.ts` gets updated.
[![butitcanbebetter](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/a40d64c1-c482-4105-8e2b-0ff82d36f113/butitcanbebetter.jpeg)](https://app.graphite.dev/macros/bobsled-inc)